src/test/compile-fail/E0254.rs

```diff
@@ -11,7 +11,7 @@
 #![feature(collections)]
 
 extern crate collections;
-//~^ NOTE previous import of `collections` here
+//~^ NOTE previous import of the extern crate `collections` here
```

Comment:
I don't mind either way.

The original behavior was to regard this as an `extern crate`, although it was labelled as such in the error message rather than the note.

I moved the label to the note as it is ambiguous whether the